pytket/qir/conversion/api.py

```diff
@@ -38,6 +40,7 @@ def pytket_to_qir(
     circ: Circuit,
     name: str = "Generated from input pytket circuit",
     returntype: ReturnTypeQIR = ReturnTypeQIR.BINARY,
+    pyqir_0_7_compatibility: bool = False,
```

Comment:
Should document the effect of this in the docstring below.

Reply:
Done in the merge commit